src/main/resources/org/jenkinsci/plugins/docker/workflow/Docker.groovy

```diff
@@ -187,6 +187,10 @@ class Docker implements Serializable {
         public String port(int port) {
             docker.script.sh(script: "docker port ${id} ${port}", returnStdout: true).trim()
         }
+
+        public String exec(String command)  {
+            docker.script.sh(script: "docker exec ${id} ${command}", returnStdout: true).trim()
```

Comment:
This could be
```
public String exec(String args = '', String command) {
    docker.script.sh(script: "docker exec${args != '' ? ' ' + args : ''} ${id} ${command}", returnStdout: true).trim()
}
```